react/Wrapper.tsx

```diff
@@ -50,27 +46,41 @@ const Wrapper: FC<Props> = ({
     [assemblyOptions, product, selectedItem, selectedQuantity, selectedSeller]
   )
 
-  const isAvailable =
-    isEmptyContext || available != null
-      ? available
-      : Boolean(
-          selectedSeller &&
-            selectedSeller.commertialOffer &&
-            selectedSeller.commertialOffer.AvailableQuantity > 0
-        )
+  const checkAvailability = (availableProp: Props['available']) => {
```

Comment:
Can this be a function defined outside of the component scope?

Reply:
Sure!